CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+Anyone is welcome to help with Firefox Accounts. Feel free to get in touch with other community members on IRC, the
+mailing list or through issues here on GitHub.
+
+- IRC: `#fxa` on `irc.mozilla.org`
+- Mailing list: <https://mail.mozilla.org/listinfo/dev-fxacct>
+- and of course, [the issues list](https://github.com/mozilla/fxa-basket-proxy/issues)
+
+## Bug Reports ##
+
+You can file issues here on GitHub. Please try to include as much information as you can and under what conditions
+you saw the issue.
+
+## Sending Pull Requests ##
+
+Patches should be submitted as pull requests (PR).
+
+Before submitting a PR:
+- Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
+  - Run `grunt eslint` to make sure your code passes linting.
```

Comment:
We should probably generic this down to `grunt lint` (which runs ESLint and JSCS; https://github.com/mozilla/fxa-basket-proxy/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R25)